datadog.go

```diff
@@ -69,6 +69,10 @@ type Options struct {
 
 	// Tags specifies a set of global tags to attach to each metric.
 	Tags []string
+
+	// TraceTags holds a set of tags that will be automatically applied to
+	// all spans.
+	TraceTags map[string]interface{}
```

Comment:
Let's call them `GlobalTags` to match the terminology in [`dd-trace-go`](https://github.com/DataDog/dd-trace-go/blob/v1/ddtrace/tracer/option.go#L79)

Reply:
fixed